### ISSUE TITLE
Integration Vuefire

### ISSUE THREAD

**Issue by chrisWolter** [Sybit-Education/COVID-Bouncer#43]
- [ ] Binding der CheckinListe (UserListe)

- [ ] Binding der Buttons

- [ ] Binding des Indikators

- [ ] Binding Indikator RaumListenSeite

**Comment by stritti** [comment]
Could lead in more reads,

**Comment by chrisWolter** [comment]
Yes, but we can probably bundle these requests so that Firebase is not requested on every page
